base-theme/assets/css/main.scss

```diff
@@ -74,7 +76,7 @@ h2 {
 
 h3 {
   font-size: 24px;
-  line-height: 2.4rem;
```

Comment:
removing this line-height, to keep the css of h3 generic.
This line-height should be targeted to wherever required